czkawka_core/src/duplicate.rs

```diff
@@ -355,109 +356,136 @@ impl DuplicateFinder {
 
         //// PROGRESS THREAD END
 
-        while !folders_to_check.is_empty() {
+        let mut bounds: Range<usize> = 0..folders_to_check.len();
```

Comment:
Why is range needed here?

Instead `bounds` and `current_slice` I used directly `folders_to_check` and it compiles fine
Only before `// Process collected data` I need to put `folders_to_check = vec![]`

Reply:
@qarmin I see: it's not needed. The new commit does `folders_to_check.clear()`. I coped a few lines of code and didn't realize it can be simpler.